static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -75,7 +75,9 @@ define([
                 },
                 shouldSendCallback: function (data) {
                     var isDev = config.page.isDev;
-                    if (isDev) {
+                    var isIgnored = typeof(data.tags.ignored) !== 'undefined' && data.tags.ignored;
```

Comment:
@piuccio I believe the Raven lib wraps it before being handed to the shouldSendCallback which adds on the extra data like tags, so I think tags will exist, but ignored is not necessarily set as one of those tags.